tests/models/animals/test_animal_model.py

```diff
@@ -23,6 +23,18 @@ def functional_group_list_instance(shared_datadir):
     return fg_list
 
 
+@pytest.fixture
```

Comment:
Yeah add this fixture to `contest.py` so you don't need to add it in multiple places (as it's a fixture you don't need to import it either)